.travis.yml

```diff
@@ -29,14 +29,13 @@ addons:
       - google-chrome-stable
   artifacts:
     paths:
-      - "$TRAVIS_BUILD_DIR/reports/"
-
+      - $TRAVIS_BUILD_DIR/reports/gemini
```

Comment:
We should also be capturing coverage, so shouldn't need to change this line

Reply:
Will revert.